guide/getting-started/14-hardware.md

```diff
@@ -20,7 +20,7 @@ Generally, any hardware with an internet connection can interact with the bitcoi
 
 ## Recovery phrase storage
 
-Simple, cheap and robust storage solutions for recovery phrases come in modest forms. [Seedplate](https://bitcoinseedbackup.com/), for example, is a metal plate in which users can etch the words of their recovery phrase for long-term offline storage. [CardWallet](https://www.cardwallet.com), in comparison, is the size of a credit card and comes pre-loaded with a recovery phrase (hidden behind a security seal) and a matching bitcoin address. And if you like an even simpler solution, a piece of paper will do. Although paper is not as durable and may be easy to loose, it is one of the strengths of Bitcoin that such simple solutions are possible.
+Simple, cheap and robust storage solutions for recovery phrases come in modest forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, for example, is a metal plate in which users can etch the words of their recovery phrase for long-term offline storage. [CardWallet](https://www.cardwallet.com){:target="_blank"}, in comparison, is the size of a credit card and comes pre-loaded with a recovery phrase (hidden behind a security seal) and a matching bitcoin address. And if you like an even simpler solution, a piece of paper will do. Although paper is not as durable and may be easy to loose, it is one of the strengths of Bitcoin that such simple solutions are possible.
```

Comment:
I'm vehemently against writing down recovery phrases but have used https://shieldfolio.com/ in the past and it's pretty good I think. Really durable and the ghost pen provides low visibility where the seeds are only visible under UV light.

Reply:
I think recovery phrases are actually one of the most powerful things about bitcoin because they allow anyone with a pen and paper (or good memory) to have custody their bitcoin. You don't even need a computer. Not ideal if you have access to better tech, but still an important aspect.

Shieldfolio looks cool, but a bit expensive for a notebook and an invisible ink pen. But definitely way cooler than if you just bought generic ones off of Amazon.